cmd/agent/subcommands/integrations/command.go

```diff
@@ -133,6 +134,10 @@ You must specify a version of the package to install using the syntax: <package>
 	installCmd.Flags().BoolVarP(
 		&cliParams.thirdParty, "third-party", "t", false, "install a community or vendor-contributed integration",
 	)
+	installCmd.Flags().BoolVar(
+		&cliParams.unsafeDisableVerification, "unsafe-disable-verification", false, "Disable TUF and in-toto integrity verification",
```

Comment:
Maybe add an additional warning in the flag description. Something like:

https://github.com/DataDog/integrations-core/blob/b83d90707bcb3029f3bead88c602278c7db24f82/datadog_checks_downloader/datadog_checks/downloader/cli.py#LL88C18-L88C18

https://github.com/DataDog/integrations-core/blob/b83d90707bcb3029f3bead88c602278c7db24f82/datadog_checks_downloader/datadog_checks/downloader/download.py#L86

Reply:
Sounds good. I'll apply @trishankatdatadog 's suggestion since it applies to this comment.